### ISSUE TITLE
allow empty `r_pkgs` in `rix::rix()`

### ISSUE THREAD

**Issue by philipp-baumann** [b-rodrigues/rix#63]
```{r}
r$> rix::rix(r_ver = "latest", git_pkgs = "nix")
Error in get_rPackages(r_pkgs) : 
  argument "r_pkgs" is missing, with no default

r$> sessioninfo::session_info()
─ Session info ───────────────────────────────────────────────────────────────────────────
 setting  value
 version  R version 4.2.1 (2022-06-23)
 os       Rocky Linux 8.7 (Green Obsidian)
 system   x86_64, linux-gnu
 ui       X11
 language (EN)
 collate  en_US.UTF-8
 ctype    en_US.UTF-8
 tz       Europe/Zurich
 date     2023-08-25
 pandoc   2.0.6 @ /usr/bin/pandoc

─ Packages ───────────────────────────────────────────────────────────────────────────────
 ! package     * version date (UTC) lib source
   cli           3.6.1   2023-03-23 [1] CRAN (R 4.2.1)
   curl          5.0.1   2023-06-07 [1] CRAN (R 4.2.1)
   httr          1.4.6   2023-05-08 [1] CRAN (R 4.2.1)
   jsonlite      1.8.7   2023-06-29 [1] CRAN (R 4.2.1)
 P pak         * 0.5.1   2023-04-27 [?] CRAN (R 4.2.1)
   R6            2.5.1   2021-08-19 [1] CRAN (R 4.2.1)
   renv          0.17.3  2023-04-06 [1] CRAN (R 4.2.1)
   rix         * 0.1.2   2023-08-25 [1] Github (b-rodrigues/rix@2a45b67)
   sessioninfo   1.2.2   2021-12-06 [1] CRAN (R 4.2.1)

 [1] /home/spectral-cockpit/git/philipp-baumann/rix_tests/renv/library/R-4.2/x86_64-pc-linux-gnu
 [2] /home/spectral-cockpit/.cache/R/renv/sandbox/R-4.2/x86_64-pc-linux-gnu/6e5b3dc8

 P ── Loaded and on-disk path mismatch.

──────────────────────────────────────────────────────────────────────────────────────────
```

**Comment by philipp-baumann** [comment]
all we need is 

```{r}
library("rix")

rix(
  r_ver = "latest",
  r_pkgs = "data.table",
  system_pkgs = "nix",
  git_pkgs = list(
    list(
      package_name = "rix",
      repo_url = "https://github.com/b-rodrigues/rix",
      branch_name = "master",
      commit = "0c735909d8db4928dbecdc9f301767a5b21bc739"
    )
  ),
  ide = "other"
)
```

**Comment by philipp-baumann** [comment]
addressed in #67
